modules/Microsoft.OperationsManagement/solutions/.test/min/deploy.test.bicep

```diff
@@ -0,0 +1,46 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.operationsmanagement.solutions-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'omsmin'
+
+// =========== //
+// Deployments //
+// =========== //
+
+// General resources
+// =================
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
+  name: resourceGroupName
+  location: location
+}
+
+module resourceGroupResources 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, location)}-paramNested'
+  params: {
+    logAnalyticsWorkspaceName: 'dep-<<namePrefix>>-law-${serviceShort}'
+  }
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../deploy.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: 'Updates'
+    logAnalyticsWorkspaceName: last(split(resourceGroupResources.outputs.logAnalyticsResourceId, '/'))
```

Comment:
Why not returning the law name from the dependencies?

Reply:
I'm usually returning the resource ID for the added flexibility it gives us.

Reply:
Personally I'd vote for simplicity, since the only purpose of the dependencies file is to return values needed for the module test. I guess we should find a third  👩‍⚖️

Reply:
Agree. Sidenote: One may argue it is "simple" that the dependencies file returns IDs consistently 😄

Reply:
I would vote for consistency and if what we're doing in most of our dependencies is using the resource ID as output then we should continue to use that.

Reply:
I guess there is no right or wrong anyways. For current latest main, most outputs are resourceIDs (because that's what we usually need), but there are cases where we also return a name. 

Those are:
- 10x Names of keys/secets in KeyVaults
- 1x the name of an MSI
- 4x the name of an AKS cluster
- 1x the name of a storage account
- 2x the name of a backup policy

So depending on what we decide, I'd vouch to align the above examples - or vice versa those cases where we used 'last(split())'.

Reply:
+1 for consistency depending on the final decision. Still on the side of returning only the data we need. Otherwise, I don't see the point of returning the resource name in all our modules given we already return the resource ID.

Reply:
Created issue #2274 & PR #2275